README.md

```diff
@@ -32,18 +32,21 @@ pip install -U pip setuptools
 pip install -e '.'
 ```
 
-Set up the `target-csv` virtual environment according to the instructions
+Like all Singer taps, the output of `tap-fixerio` should be piped to a Singer target. 
+Deactivate the `tap-fixerio` virtual environment by using the `deactivate` command, 
+then set up the `target-csv` virtual environment according to the instructions
 [here](https://github.com/singer-io/target-csv/blob/master/README.md).
-These commands will install `tap-fixerio`  with pip, and then run it:
+
+Once `target-csv`, or another Singer target, is installed in its own virtual environment run them with the following command:
```

Comment:
This line should be hard-wrapped like the above paragraph.

Reply:
Done!